src/CMake/dkms.cmake

```diff
@@ -184,6 +184,8 @@ SET (XRT_DKMS_DRIVER_SRCS
   xocl/subdev/xdma.c
   xocl/subdev/qdma.c
   xocl/subdev/qdma4.c
+  xocl/subdev/stmc.h
+  xocl/subdev/stmc.c
```

Comment:
Looks like stmc is a new subdev. Why are not the subdev id and name defined in devices.h?

Reply:
I have added NODE_STM4 in device.h in the previous check-ins.
Do i need to rename these 2 files to stm4.[ch]?